decidim-core/app/models/decidim/messaging/message.rb

```diff
@@ -44,11 +44,23 @@ def envelope_for(recipients)
         recipients.each { |recipient| receipts.build(recipient: recipient) }
       end
 
+      # Public: Returns the comment body sanitized, sanitizing HTML tags
+      def sanitized_body
```

Comment:
please, remove this mehod, we only want `body_with_links`

Reply:
Done